src/parameterStore.ts

```diff
@@ -21,11 +11,10 @@ export async function getParameter(parameterName: string): Promise<string> {
             })
             .promise();
     } catch (err) {
-        logger.error('ParameterStore error:', err);
+        console.error('ParameterStore error:', err);
         throw err;
     }
     const parameterValue = result.Parameter!.Value!;
-    logger.info(parameterName, parameterValue);
```

Comment:
logger is empty implementation, it doesn't log anything.

Reply:
Not sure I understand.  Are you saying that the call to makeLogger returns an object with an info method that doesn't do anything?  Doesn't the rest of the FHIR Works project use this type of logger?

Reply:
it doesn't do anything. I couldn't find anywhere in the project that uses logger. All of them are `console`